test/bindings/http/promiscuous_receiver_test.js

```diff
@@ -0,0 +1,90 @@
+const { expect } = require("chai");
+const { CloudEvent, HTTPReceiver } = require("../../../index.js");
+const constants = require("../../../lib/bindings/http/constants.js");
+
+const receiver = new HTTPReceiver();
+const id = "1234";
+const type = "org.cncf.cloudevents.test";
+const source = "urn:event:from:myapi/resourse/123";
+const data = {
+  lunch: "sushi"
+};
+
+describe("HTTP Transport Binding Receiver for CloudEvents", () => {
+  describe("V1", () => {
+    const specversion = "1.0";
+
+    it("Structured data returns a CloudEvent", () => {
+      const payload = {
+        id,
+        type,
+        source,
+        data,
+        specversion
+      };
+
+      const headers = {
+        "Content-Type": "application/cloudevents+json"
+      };
+
+      const event = receiver.accept(headers, payload);
+      validateEvent(event, specversion);
+    });
+
+    it("Binary data returns a CloudEvent", () => {
+      const headers = {
+        "Content-Type": constants.DEFAULT_CONTENT_TYPE,
+        [constants.DEFAULT_SPEC_VERSION_HEADER]: specversion,
+        [constants.BINARY_HEADERS_1.ID]: id,
+        [constants.BINARY_HEADERS_1.TYPE]: type,
+        [constants.BINARY_HEADERS_1.SOURCE]: source
+      };
+
+      const event = receiver.accept(headers, data);
+      validateEvent(event, specversion);
+    });
+  });
+
+  describe("V03", () => {
+    const specversion = "0.3";
+
+    it("Structured data returns a CloudEvent", () => {
+      const payload = {
+        id,
+        type,
+        source,
+        data,
+        specversion
+      };
+
+      const headers = {
+        "Content-Type": "application/cloudevents+json"
```

Comment:
Like above, perhaps use the constants:
https://github.com/cloudevents/sdk-javascript/blob/e329d9ac00d6a2e8a29d0aca6d18d7c4eddaa02e/lib/bindings/http/constants.js#L20
https://github.com/cloudevents/sdk-javascript/blob/e329d9ac00d6a2e8a29d0aca6d18d7c4eddaa02e/lib/bindings/http/constants.js#L18